urls.py

```diff
@@ -36,7 +40,18 @@
         'https://s3-us-west-2.amazonaws.com/qiime2-data/distro/core/qiime20174-1492814818.zip',
     'distro/core/qiime20174-1492814818.zip':
         'https://s3-us-west-2.amazonaws.com/qiime2-data/distro/core/qiime20174-1492814818.zip',
-    
+
+    # 2017.5 DISTRO
+    'distro/core/qiime2-2017.5-conda-osx-64.txt':
+        'https://raw.githubusercontent.com/qiime2/environment-files/master/2017.5/release/qiime2-2017.5-conda-osx-64.txt',
+    'distro/core/qiime2-2017.5-conda-linux-64.txt':
+        'https://raw.githubusercontent.com/qiime2/environment-files/master/2017.5/release/qiime2-2017.5-conda-linux-64.txt',
+    ### TODO: Update these URLs
```

Comment:
Discussed with @jairideout & @ebolyen: we will continue to provide VBox downloads as data.qiime2.org links for now, but we will probably revisit in the future!